docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
   - /topics/storage.md
 ---
 
-TODO: update per v2 (no longer any job cache)
+# Storage in Brigade
 
-# How Brigade uses Kubernetes Persistent Storage
+Brigade utilizes storage for its in the following ways:
 
-Brigade allows script authors to declare two kinds of storage:
+  * [Shared Worker storage](#shared-worker-storage) wherein a Brigade Worker's
+    workspace may be shared with its Job(s). Optional; not enabled by default.
```

Comment:
Nit: The way this is worded, it isn't clear that jobs can use this to share information with one another.

Reply:
Good point... what about `may be shared among its Job(s)`?

Reply:
How about "with _and_ among?"